test/integration/test.ts

```diff
@@ -85,12 +85,22 @@ suite("Launch and onboarding", function() {
     await app.client.click(".QueryList-new i");
     setValueToEditor("select 1;");
     await app.client.click("ul.QueryList-list li:last-child");
+    await app.client.waitUntil(
+      async () => (await getValueFromEditor()) === "select * from data_sources",
+      5000,
+      "Timeout 1",
```

Comment:
Failed Travis CI says "Promise timeout", not "Timeout 1" (this string).
Maybe Promise execution is timeout within waitUntil default interval (250msec).
I extend to 1000 msec.